x-pack/filebeat/module/oracle/database_audit/_meta/fields.yml

```diff
@@ -0,0 +1,70 @@
+- name: database_audit
+  type: group
+  release: beta
+  description: >
+    Module for parsing Oracle Database audit logs
+  fields:
+    - name: status
+      type: keyword
+      description: >
+        Database Audit Status.
+
+    - name: sessionid
+      type: keyword
+      description: >
+        Session ID related to the audit event.
+
+    - name: client_terminal
+      type: keyword
+      description: >
+        If available, the client terminal type, for example "pty"
+
+    - name: client_address
+      type: keyword
+      description: >
+        The IP Address or Domain used by the client.
+
+    - name: client_user
+      type: keyword
+      description: >
+        The user running the client or connection to the database
+
+    - name: database_user
```

Comment:
`database.user` probably better than `database_user` in case we improve the module in the future to extract more "database related" fields